File: decompiled/sources/com/hjq/permissions/PermissionDelegate.java
package com.hjq.permissions;

import android.app.Activity;
import android.content.Context;
import android.content.Intent;

public interface PermissionDelegate {
    Intent getPermissionIntent(Context context, String str);

    boolean isGrantedPermission(Context context, String str);

    boolean isPermissionPermanentDenied(Activity activity, String str);
}

File: decompiled/sources/com/hjq/permissions/PermissionDelegate.kt
package com.hjq.permissions

import android.app.Activity
import android.content.Context
import android.content.Intent

interface PermissionDelegate {
    fun getPermissionIntent(context: Context, permission: String): Intent

    fun isGrantedPermission(context: Context, permission: String): Boolean

    fun isPermissionPermanentDenied(activity: Activity, permission: String): Boolean
}
